keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -754,6 +736,9 @@ in back-to-back releases.
 
 #### GA
 
+- `kubectl` environment variables and API Server feature gates are locked to on by default.
+- Deprecate `kubectl` environment variables and API Server feature gates for future removal.
+- Address RBAC authorization for WebSocket upgrades. The mechanism must be compatible with existing authorization rules for subresources (e.g., `pods/exec`) that are typically enforced on POST requests.
```

Comment:
@liggitt for your comment in https://github.com/kubernetes/kubernetes/issues/133515#issuecomment-3229672099 and your code there is unclear to me if you want to rollout this authoritzation compatibility as a separate gate or making this GA but not locked by default during one cycle.

Reply:
1. I have updated the KEP to indicate GA in 1.36.
2. I have added a separate feature gate `ForceRBACCreateCheck` for the additional `CREATE` authz check. This functionality is a prerequisite for GA.

Please let me know what you think.